app/lib/widgets/gallery/gallery.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flex_ui/flex_ui.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+
+class FlexGallery extends StatefulWidget {
+  const FlexGallery({
+    super.key,
+    this.imageUrls = const [],
+    this.thumbnailUrls = const [],
+    this.borderRadius = 0,
+  });
+
+  final List<String> imageUrls;
+  final List<String> thumbnailUrls;
```

Comment:
Just a thought I had when implementing,
Could probably default to just one list of imageUrls, and allow thumbnailUrls as optional in case user also has a list of thumbnail specific urls as well.

Both lists need to be same length, and in the same order or you have issues
(2 images, 3 thumbnails)
<img width="432" alt="image" src="https://github.com/user-attachments/assets/c22cddee-d0ab-498e-a07f-eaa2edf98c5a">

Simplifies base use-case but Ultimately implementer responsibility - so totally non blocking